packages/cjs-module-analyzer/index.js

```diff
@@ -7,6 +7,7 @@
 /* eslint no-continue: ["off"] */
 /* eslint no-shadow: ["off"] */
 /* eslint no-unreachable-loop: ["off"] */
+/* eslint @endo/restrict-comparison-operands: ["off"] */
```

Comment:
cc @naugtur Would like to know whether we can turn this one on maybe someday.

Reply:
I don't see any `==` comparisons in there. What am I missing?

Reply:
>=